contracts/tests/test_validator_auction.py

```diff
@@ -158,12 +159,34 @@ def test_bidding_auction_ended(started_validator_auction, accounts, chain):
 
 
 @pytest.mark.slow
-def test_enough_bidders_ends_auction(almost_filled_validator_auction, accounts):
+def test_enough_bidders_ends_auction(
+    almost_filled_validator_auction, accounts, web3, number_of_auction_participants
+):
+
+    assert_auction_state(almost_filled_validator_auction, AuctionStates.Started)
 
     almost_filled_validator_auction.functions.bid().transact(
         {"from": accounts[1], "value": 100}
     )
 
+    assert_auction_state(almost_filled_validator_auction, AuctionStates.DepositPending)
+
+    deposit_locker = almost_filled_validator_auction.functions.depositLocker().call()
+    assert web3.eth.getBalance(deposit_locker) == 0
+
+    # we could test the rest of this in another function, but it's slow anyway
+    # and I'm lazy
```

Comment:
:(

Reply:
The test goes through the process of ending an auction. I fail to see the benefits of creating two tests out of this, other than satisfying the 'a test should only test one thing rule', which I'm willing to break. 

If you think this must be changed, please a) say so (a sad smiley is not enough) and b) state the reasons why.